torchgeo/datasets/mapinwild.py

```diff
@@ -0,0 +1,491 @@
+"""MapInWild dataset."""
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+from collections.abc import Sequence
+from typing import Any, Callable, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
```

Comment:
Pandas is currently an optional dependency, it won't always be installed. You need to use a lazy import instead of importing it at the top level. See any of these datasets for examples of what this looks like:
```console
> grep -l pandas *.py
eddmaps.py
gbif.py
idtrees.py
inaturalist.py
openbuildings.py
reforestree.py
usavars.py
western_usa_live_fuel_moisture.py
```
This is why the Read the Docs tests are failing.

Reply:
Thanks for the heads up!